pkg/staticnetworkconfig/generator_test.go

```diff
@@ -0,0 +1,34 @@
+package staticnetworkconfig
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+func TestStaticNetworkConfig(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "StaticNetworkConfig Suite")
+}
+
+var _ = Describe("StaticNetworkConfig", func() {
+	var (
+		staticNetworkGenerator = StaticNetworkConfigGenerator{log: logrus.New()}
+	)
+
+	It("hw interface name validation", func() {
+		mac, err := staticNetworkGenerator.validateAndCalculateMAC("09230fd892AA")
```

Comment:
pls add a case for a shorter than 12 chars mac-address.

there is an option to use data-driven test - https://onsi.github.io/ginkgo/#table-driven-tests
which will allow also to provide a description of each test(mac too long, mac too short, illegal mac character)

Reply:
added